_docker/tests/test-runner/test_run_e2e_tests_options.rb

```diff
@@ -113,12 +113,12 @@ def test_docker_execution_specifying_update_github_status
 
   def test_docker_execution_specifying_mobile_device
 
-    test_configuration = @run_tests_options.parse_command_line(%w(--e2e --base-url=http://foo.com --use-docker --browser=iphone_6))
+    test_configuration = @run_tests_options.parse_command_line(%w(--e2e --base-url=http://foo.com --use-docker --browser=iphone_8))
 
     assert(test_configuration[:docker])
-    assert_equal('iphone_6', test_configuration[:browser])
+    assert_equal('iphone_8', test_configuration[:browser])
```

Comment:
Is the latest version of the phone (yes, I know the X is out) what we want to be testing against all the time? Seems a bit bleeding edge to me, but maybe there's a good reason